src/main/java/net/sf/jabref/external/ExternalFileTypeEntryEditor.java

```diff
@@ -184,10 +181,16 @@ public void changedUpdate(DocumentEvent documentEvent) {
             });
         }
 
+        String title = "Edit file type";
+        
+        if(entry.getName().isEmpty()) {
+            title = "Add new file type";
+        }
+        
         if (dParent != null) {
-            diag = new JDialog(dParent, Localization.lang("Edit file type"), true);
+            diag = new JDialog(dParent, Localization.lang(title), true);
```

Comment:
We have a script that automatically extracts the used language keys. This only works when we do not pass a variable to the `lang` method. Can you please inline the language keys as Strings?